src/client/app/components/admin/PreferencesComponent.tsx

```diff
@@ -308,8 +401,13 @@ export default function PreferencesComponent() {
 					type='number'
 					value={localAdminPref.defaultFileSizeLimit}
 					onChange={e => makeLocalChanges('defaultFileSizeLimit', e.target.value)}
+					min='0'
```

Comment:
How about changing this to ``min={Number(localAdminPref.defaultWarningFileSize)}`` so it more consistent with other min/max pairs and enforces the min limit?

Reply:
I also modified the function to be consistent. I forgot to mention that.